src/txn/txn_rollback_to_stable.c

```diff
@@ -111,6 +111,10 @@ __txn_abort_newer_update(
             upd->durable_ts = upd->start_ts = WT_TS_NONE;
         }
     }
+
+    /* Reset the history store flag for the stable update. */
+    if (first_upd != NULL)
+        F_CLR(first_upd, WT_UPDATE_HS);
```

Comment:
Is this to make sure the stable update is written to history?  Can there be more updates with the flag set, and if so, is that okay?  Ideally this would be explained in the comment: at the moment it is describing what the code does, but not why.

Reply:
We are clearing the flag to indicate that this update is not moved into the history store. I added a comment to explain. Is that comment provides better understanding?